pyproject.toml

```diff
@@ -18,9 +18,9 @@ classifiers = [
 
 
 dependencies = [
-    'numpy',
-    'fasm',
-    'docker',
+    'python-dotenv>=1.0.0',
```

Comment:
Could you please remove the version pinning in pyproject.toml?

Usually, you only pin the versions in the requiremtens.txt, for deployable/standalone applications. In the pyproject.toml you only add unpinned dependencies, that you don't get any dependency versioning troubles when importing the project as a dependency itself.

Reply:
Hi, I only added a lower boundary as shown in the user guide here: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#dependencies-and-requirements
If dependencies have versions lower than that, then FABulous probably won't work anyway.

Nix's buildPythonPackage looks in pyproject.toml for the dependencies. If I remove the version range from there, I wouldn't get a warning if the dependencies don't have the right versions.

Reply:
Sorry, I overlooked that, I wasn't fully awake yet. :sweat_smile: 
As long as it's only the lower limit, it should be fine. :+1:

Reply:
No worries, thanks for the review 😃